cmd/ocm-backplane/console/console_test.go

```diff
@@ -104,17 +111,18 @@ var _ = Describe("console command", func() {
 			CurrentContext: "default/testcluster/testauth",
 			Extensions:     nil,
 		}
+		
```

Comment:
You need to initialize the `clusterInfo` here
```suggestion
		clusterInfo, _ = cmv1.NewCluster().
			CloudProvider(cmv1.NewCloudProvider().ID("aws")).
			Product(cmv1.NewProduct().ID("dedicated")).
			AdditionalTrustBundle("REDACTED").
	            Proxy(cmv1.NewProxy().HTTPProxy("http://my.proxy:80").HTTPSProxy("https://my.proxy:443")).
			Build()
```

Reply:
@reetika-vyas was this added? I do not see it in the PR.